### PR TITLE
RTL support added to ToolbarAndroid via "rtl" attribute.

### PR DESCRIPTION
**Pull request by lxsameer**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native",
-  "version": "0.12.0",
+  "version": "0.17.0-rc",
```
Comment (satya164) [inline package.json]: Could you revert this?

Comment (lxsameer) [inline package.json]: Yes, but what should be the version number ? react native 0.16 is out but master version is 0.12.0

Comment (ide) [inline package.json]: The version number is set by each release. You should send PRs to master.

Comment (lxsameer) [inline package.json]: Ah sure. But we need to change the contribution guide too clearly say it. That guide caused this.

**Comment by lxsameer** [comment]
I'm closing this PR in favor of #4378

**Pull request closed without merging**